astroquery/irsa/core.py

```diff
@@ -111,7 +111,11 @@
 from ..query import BaseQuery
 from ..utils import commons
 from . import conf
-from ..exceptions import TableParseError, NoResultsWarning
+from ..exceptions import (
```

Comment:
no need for the linebreak here, we allow longer than 80 char lines, up to 100-120 is fine.

Reply:
Great, I'll adjust this.